capstone/src/main/java/com/step902020/capstone/ViewController.java

```diff
@@ -0,0 +1,61 @@
+package com.step902020.capstone;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class ViewController {
+
+  @GetMapping("profile")
```

Comment:
These mappings are not done automatically? 
This is how you'd customize matching between URLs and views, but I am curious that the default is not Just Working: https://docs.spring.io/spring/docs/current/spring-framework-reference/web.html#mvc-config-path-matching